src/js/view/controls/controlbar.js

```diff
@@ -441,11 +441,15 @@ export default class Controlbar {
 
     // Close menus if it has no event.  Otherwise close all but the event's target.
     closeMenus(evt) {
-        this.menus.forEach(ele => {
-            if (!evt || evt.target !== ele.el) {
-                ele.closeTooltip(evt);
-            }
-        });
+        if (!evt) {
```

Comment:
let's reduce the nesting with an early return statement. I find early returns easier to read and nest if statements tend to get tangly. Also you don't need that `else` statement.
```
if (evt) {
        return;
}
this.menus.forEach(ele => {
            if (evt.target !== ele.el) {
                    ele.closeTooltip(evt);
            }
});
```